### ISSUE TITLE
Cannot remove BGP redistribute route map

### ISSUE THREAD

**Issue by dicristina** [fortinetdev/terraform-provider-fortios#294]
It seems that it is impossible to remove a redistribute route map from the `fortios_router_bgp` resource once it is set.  Example:

```terraform
resource "fortios_router_bgp" "bgp" {

  as = var.AS

  redistribute {
    name   = "connected"
    status = "enable"
    route_map = "myRouteMap"
  }

}
```

The following actions do **not** cause a change to the  `fortios_router_bgp` resource:

- Removing the `route_map` attribute
- Setting `route_map = null`
- Setting `route_map = ""`

The redistribute route map should be unset if the `route_map` attribute is removed.

### Versions
Provider: 1.17.0
Terraform: 1.5.5
FortiOS: 7.2.5

**Comment by MaxxLiu22** [comment]
Hi @dicristina ,

Thank you for raising this issue, I can reproduce it on my side, here is a workaround solution for temporary use, I have reported it to the development team, Thank you for your help.

```
resource "fortios_json_generic_api" "unset" {
  path   = "/api/v2/cmdb/router/bgp"
  method = "PUT"
  json   = <<EOF
{
      "redistribute": [
    {
      "name": "connected",
      "status": "enable",
      "route-map": null
    }
  ]
}
EOF
}
```

Thanks,
Maxx

**Comment by MaxxLiu22** [comment]
Hi @dicristina ,

This issue has been fixed in our latest Terraform FOS 1.19.0,``route-map: null`` would unset this argument. I will go ahead to close this case, you are free to reopen it or another case if you have any other questions.

Thanks,
Maxx
